Dockerfile

```diff
@@ -33,10 +33,11 @@ ENV MOJO_REVERSE_PROXY 0
 
 # Do not change these variables unless you know what you're doing
 ENV CONVOS_HOME /data
-ENV MOJO_LISTEN http://*:3000
+ENV MOJO_LISTEN http://*:$port
```

Comment:
It does not. Unless you want to run another mojo app in this container, which of course you don't.